app/Support/SearchIndexBuilder.php

```
@@ -394,6 +403,75 @@ private static function buildLegacyCondition(\mysqli $db, string $column, string
+     * (buildSearchCondition) decides WHICH rows qualify; this decides their
+     * order. Per query word a hit scores: title 100, author 60, subtitle 40,
+     * keywords 10, description 3 — summed across words — then rows are ordered
+     * by that score DESC with title ASC as the tie-break.
+     *
+     * The returned params must be appended to the statement AFTER the WHERE
+     * params (the ORDER BY placeholders come last) and the types concatenated
+     * after the WHERE types.
+     *
+     * @return array{sql: string, params: list<string>, types: string}
+     */
+    public static function buildRelevanceOrder(\mysqli $db, string $searchQuery, string $prefix = 'l.'): array
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $db. Configured minimum length is 3. (undefined)

(ShortVariable)